### PR TITLE
New zero_trigger implementation with unit test

### PR DESCRIPTION
**Pull request by Nitnelav**

### CODE REVIEW

File: services/rpi_systemd/zerotrigger_config.json [hunk 1]
```diff
@@ -5,7 +5,9 @@
   ],
   "configuration_file": "",
   "trigger_tag": ["Train", "Rail transport", "Railroad car, train wagon"],
```
Comment (nicolas-f) [inline services/rpi_systemd/zerotrigger_config.json:7]: it is trigger_tags now with your modification

**Review by nicolas-f — CHANGES_REQUESTED** [review]
need to update trigger_tags setting file

File: services/rpi_systemd/zerotrigger_config.json (hunk 1 above)
Comment (Nitnelav) [reply services/rpi_systemd/zerotrigger_config.json:7]: done

Comment (nicolas-f) [reply services/rpi_systemd/zerotrigger_config.json:7]: the file has not been fixed

**Review by nicolas-f — APPROVED** [review]

**Pull request merged**